### ISSUE TITLE
Model Code

### ISSUE THREAD

**Issue by rowanc1** [simpeg/simpeg#71]
To create a block model for example:

```
blkModel = Models.Block(mesh)
blkModel.p0 = [1,2,3] # can supply these as kwargs
blkModel.p1 = [4,5,6]
blkModel.prop = [1e-2,1e-4]
cond = blkModel.transform
```

The block model should also have a derivative associated with it.

**Comment by lheagy** [comment]
this should be a mapping?

**Comment by rowanc1** [comment]
#471 with the `Wires` is a first crack at this. It is not possible to set these yet though. We can look at this maybe in #320?

**Comment by lheagy** [comment]
I think we can close this. The general approach for including parametric maps was addressed in #480. 

Thoughts @rowanc1? @sgkang?

**Comment by lheagy** [comment]
I am closing this as there hasn't been activity in a long time. Please re-open if you would like to continue discussing.
